CppcheckTargets.cmake

```diff
@@ -125,6 +117,11 @@ function(add_cppcheck _name)
   if(NOT TARGET cppcheck_${PROJECT_NAME})
     add_custom_target(cppcheck_${PROJECT_NAME})
   endif()
+  if(NOT TARGET ${PROJECT_NAME}_tests)
```

Comment:
and CpplintTargets.cmake does not do it. I would remove it from here, or else it must also be added in the CpplintTargets.cmake...